source/_integrations/home_connect.markdown

```diff
@@ -96,6 +103,7 @@ Sets an option for the selected program.
 
 ### Service `home_connect.pause_program`
 
+This service is deprecated. Please use the button for this operation.
```

Comment:
Please remove deprecated documentation instead of marking it deprecated.

Thanks 👍

Reply:
Even if the code is still there and exists?

Reply:
removed